src/ReactLeafletMap/layers/SvgVectorLayer/GeoJsonLayer/index.js

```diff
@@ -79,11 +79,41 @@ class GeoJsonLayer extends React.PureComponent {
         })
     }
 
+	/**
+	 * Set style of the feature
+	 * @param leafletStyle {Object} Leaflet style definition
+	 * @param element {Object} Leaflet element
+	 */
+	setStyle(leafletStyle, element) {
+    	const shape = element?.options?.icon;
+    	if (shape) {
+    		shape.setStyle(leafletStyle, shape.id, shape.isBasicShape);
```

Comment:
Indetention

Reply:
It'll be fixed in the next PR when our code styling rules are applied